### ISSUE TITLE
Why do you leave out batch norm on the first iteration of creating the discriminator for GAN.py?

### ISSUE THREAD

**Issue by MLSlayer** [davidADSP/GDL_code#22]
It kind of struck my curiosity on why you left out Batchnorm on iteration 0.

https://github.com/davidADSP/GDL_code/blob/master/models/GAN.py#L105

**Comment by satya400** [comment]
Hi @MLSlayer / anyothers , could you find any information on why the BN is omitted in 0'th iteration?
